src/style.css

```diff
@@ -0,0 +1,27 @@
+body,
+div {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url("images/star.svg");
+  background-size: contain;
+  background-repeat: no-repeat;
+}
```

Comment:
You need some space between stars. Try to add margin ;)

Also I don't think you will need `background-size`, `background-position: center;` will do the trick.